cachier/cores/base.py

```diff
@@ -9,6 +9,8 @@
 import abc  # for the _BaseCore abstract base class
 import inspect
 
+from ..config import _Type_HashFunc
```

Comment:
Is it confusing, perhaps, to import custom types from `config`? What about having a separate `custom_types` module, that both config and other modules can import from?

Reply:
That is fine too

Reply:
done